scripts/symbolize.py

```diff
@@ -35,10 +35,16 @@
 import sys
 
 TA_UUID_RE = re.compile(r'Status of TA (?P<uuid>[0-9a-f\-]+)')
-TA_INFO_RE = re.compile(':  arch: (?P<arch>\w+)  '
+TA_INFO_RE = re.compile('  arch: (?P<arch>\w+)  '
                         'load address: (?P<load_addr>0x[0-9a-f]+)')
 CALL_STACK_RE = re.compile('Call stack:')
-STACK_ADDR_RE = re.compile(r':  (?P<addr>0x[0-9a-f]+)')
+
+# The STACK_ADDR_RE shall match lines starting with what has been stated in the
+# function trace_level_to_string(...), i.e., this is the UEIDF below, then we
+# have the thread ID and either TA or TC, finally we have the actual address. A
+# line typically look like this:
+# E0TC  0x1234abcd
```

Comment:
What if one get dozen of thread. I thought about the big multi-core presented at last connect.

Reply:
Use two digits (and `//` when no thread id)?